app/templates/me/index.hbs

```diff
@@ -31,8 +31,10 @@
 
     <p>
         If you want to use package commands from the command line, you will need to
-        login with <code>cargo login</code> using one of the tokens listed below. <br/>
-        When working in shared environments, supplying the token on the command line could expose it to other users, so it is more secure to supply a token when prompted. 
+        login with <code>cargo login (token)</code> using one of the tokens listed below. <br/>
+        When working in shared environments, supplying the token on the command line could 
+        expose it to prying eyes via command line history. To avoid this, enter <code>cargo login</code>
```

Comment:
This is just a simple hint. I'm not sure this is the place to give people a complete list of how their command line parameters can be hijacked by people who have permissions for `ps -a`. I'm concerned that too much wording/documentation here is just going to look like noise and people won't read it.